Anytype/Sources/Models/Documents/Document/BaseDocument.swift

```diff
@@ -222,7 +222,7 @@ final class BaseDocument: BaseDocumentProtocol {
     }
     
     private func setupSubscriptions() async {
-        syncStatusStorage.statusPublisher(spaceId: spaceId).sink { [weak self] info in
```

Comment:
Please check dev. I remove async, because it make a race for subscriptions array